### ISSUE TITLE
FITB accuracy only 10% of what the readme says

### ISSUE THREAD

**Issue by highintothesky** [AlenUbuntu/Fashion-AI#1]
as it says in the title, the FITB accuracy reported when running the code is very low:

```
Compute adjacency matrix up to 1 degrees, i.e., A^1
Test Acc on link prediction: 0.8361
Overall FITB Acc: 0.0979
```

your README claims a FITB accuracy of more than 90%, so more than 0.9. this is way off. i believe i installed everything correctly (your install instructions and required dependencies are missing), grabbed the dataset using your script, performed the feature extraction, used the polyvore helper script to split everything, ran the FITB test with the pretrained model provided.

any idea what could be causing this? is there something wrong with your code base? I got no errors or warnings. using python 3.6.9 with pytorch 1.6.0.
